packages/components/src/__future__/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./ExampleComponent"
```

Comment:
We can delete the `ExampleComponent` now 😄

Reply:
Yay deleting things! 🕺